src/main/resources/citizen/templates/document/countyCourtJudgmentByAdmission.html

```diff
@@ -0,0 +1,226 @@
/J5Lms/tnIa8k3f3wxN27R3Y3gGUrOnGysj0aj27k54HDDdkWv40Vwq9abEkFyc5SI0sx7DmRwFMaA5aX57YuCqnBN0lxCJVI0gW4DPRT429dESaVjeAq6ChBgoRPaq6arMnH1yrni912kOtLOWjkj7c2GtvYuJVBnUR2Va9X2KgJSvM74QZODvLOLSogjnGO6ejXnbvATxxTDx156GHnY0di7RmCjI/OtnxNw2EEYa/Br98tPyyqlCOFQCTQtS87HabhwLb/UGQBvgoQYvwQ7W/hnsQD7wozPfRsO+CymCJ4V1+Kz97Eqsf9Vrv7pEpk2Cl6FuwoqNiGecb2csUFYVBXi4FNvrOljneoeclXIBI7GrtIDMJrVrbocs7Sp92vqex/78uRKIxcqDqstgXstR+TMyaug/4OkbAv8SipZDG2RCPp3bZaEOlUiPUEU9OfIFfnzVFbQYMjldXzl2jp+IpMXKy7tSuqnusx0yWQqHd31evztNec8VPopi5W3saJUgEkJMDIzHngyghODL+3vOFQXaGWl6D6rsdH2RWuPQmr1JbYj69gdrpAdnJBGBnKtfSSCkMpqBQpYq+EBEIgiBjovpVRiLKixoSilCPKIENoREf2m42USuBMXp6LsfbKgvOz5VFNzoc6bwwySc58dt11nBiopeKEwaiGJpgOfcFgz2z94z0URLsyapYsgUJIACBZTEZtyQvpLYxZtsa3ZFpdUt1ajMrf1O5DZEHMBvbhiAu0DZaQSz9bhldqAB0RwphHouA9tFwKNZBa+sXABpHLq8EBHJVRoEz5gc0cuO4Z8qvxA2RPlNv/VyZVNdLXmty25i2iPbcVRQjFY1dG+Xo/vsK3bDuY0P7hKi9b2HS0+sloJdNOb257e52UCtxGdxGRNAASHTX/XKOyTUQfpt+xUqblS/0ZgUheKDNtr/3w4XeicTRJbwsXabhmTe2d9dXInUlG/v52bzG5mhe+y2t+XYd4kyl8BPHcmdNRPrtsHprDXGSBF5SR2k94+UGvi3cRrPUhCswvbF3YkstzYdY/0XLybiFsAxW2xq756lbwiDcYr9GEz35xcOVZVQHhhaO/NgXVhDe1hfUHfJQOFxgZc+m8wSPNLm9oNm2od/IZa2GvIFbX8rcwwPrgVoVQaU5Ipbw2ZL9KC9sR6heMZJSuqR+zcQHFHPPhw8BhJrXc6RRXDO6BRPDTkdNyXk27dQm/2wH4ifCiV9DcnlwlmuUtUAtvbvMBN+bbA8utlw1SINpiV7ClyfcNKe2K1m2ZPVr2FpM54W1V190MMT2cDNzfGlKd0lwzU50acgk8pSFvZAjuTXj20lKXVtuP7WkMU1Nu0jjAJWVHwWjAlX8S+OLlliTctdbb7MkU+bySWO97Gdref6W5GeYW5mjiH8Nf5QenyIK46qKoUoiOHGgOtnDQcsg0MLhfdDCn9s+7pLCQyLWpVO7G3iuYIdZOGgg8I7DlZ0/J0avOIv94mtljRoEQnxwmzxYR6jH1+cutCHhsFIEf0W5CRALGh6U2yIN1JLG7F2zAqEV7zi+vVA5+tUVC29xu6U6n1yZKVYR2TgnpAPKRZGovH7TJt9zn2d18ksm2Q3FE5Bf3FIm8R7Bk19hnSd4rtmE2NsGZsVIadP7Q/P7kfNoyymO042jEZXM0Iao6j8xFbIx/dfDDGNAUPrfMhIRzK3XHdx3/Ltva1Y/A8vpXaKdicnpV/jExDSWzcsR+80MjNy8rHJghxuYeq0roIJKckybX8gW6WVKIfn1zp3iQV5rOTw+04sMHDjLYKYEozG/UFhkCCJD6iL409CSg4cvoNPYL+O5Lr/bgd8vtNHzpJar2V/ey677u0xXO1FtN49Wa4t87tHHBdBcaLhqGcDoremx+f3GOrIl62kTOX3P5GCszuQhafusYtw0VBt8g7CeSwm6dfX4Eva5eu9K66xgT/kxuanf8Vi0YZx/6j5he9plvpm40HaXMHTLVmA0zRhaDNl3yzQuTK0jma+aP16cmd9mONunb1e1TBOXFTifxzaSvI44AnI0tbp0wkD8UaKnNb66nC7s4TbMv/35+g0XKby8PIqV8Nn21iK0K9xilSmraTV3LBVQyqwFkaQYGig1LwvR0fynwhwMq39qcnd9cHle8wEd8vEGnq0FG8M6AUTHi4doUcf9nrRBU5v+kRKI8E30Z/RFZk1vxXW1vhegd8vi9ngNtZnxz2IKltKAAoPEWLDJEvBAtLYH3E8yxRr61RWCk0thKLVnfGh/9lvSiExViz8b873ggZ/jP7QOoClP8vDlM0Wo3Y/Zr6LKI7udvMutHPev511VudP1lqkAxPzrzeEvWn6XY3hI+OXOMzl+BMVgSGMxVOsZlziX72nisNfZTk7DoA0VKQyfKwFOwYv0Z6NBxFtWCO0P+foL++tk2054O0ufzvZNYdfseF1Yt8TP6l7Y2X9Xhiav4XL/t+I6d+ElsUDu+w/ha+JXlMfHZyR9plgzgB2mM1BJXIKhZWAIbJ/5tW+kVwMDW2/6Tk0KglI6g0ePW98qyHUUgGYgnRTRxSV33/EASLLF2+f+QK3bXjz0FdtUf3NHl2Vfnnq47J9xQipyoUZVY6FA6h/Swl8/nJbQLXHTZAaXtylMmqc/GnkikXgoiJ/i9cww7q9id84gfGztV6cZpDn9+lsf/3QRLVpRupTHCntw37RtVdg8JY9N61jkcojl5hZKdmtAeTV4GBqmEmWCMczS97tAAWZAl6Z310coeNTaGy5WtZymS1sfWHcm9FxW62ik24omSdsN6afe0JnTbOG0XWlZbX3YaCYyNZ9CoG7qSYRw54+Rra9yPRIWfqjmYfK4o6UL3WlTtanBVbe9R83MR1awlEiRm1F4e1+RqBopP54W+oTvNFu+M2A9UMLbLxTxtqdeJCxzTL7ymmql/HvLUOYR45xf9105HPlUzJObcuq+HYtbtDVkuufRSaBSvi0wi1zl/+POiwqfYy8l9vTGsQticYwA
```

Comment:
The logic to display more contents for payment `instalments` and `set date` are missing. please add